types/tween.js/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for tween.js 17.2
+// Type definitions for tween.js 18.5
 // Project: https://github.com/tweenjs/tween.js/
-// Definitions by: jordan <https://github.com/Amos47>, sunetos <https://github.com/sunetos>, jzarnikov <https://github.com/jzarnikov>, alexburner <https://github.com/alexburner>
+// Definitions by: jordan <https://github.com/Amos47>, sunetos <https://github.com/sunetos>, jzarnikov <https://github.com/jzarnikov>, alexburner <https://github.com/alexburner>, olafurkarl <https://github.com/olafurkarl>
```

Comment:
this is minor, but woudlbe nice if we could finally split this into new lines:
https://github.com/definitelytyped/definitelytyped#edit-an-existing-package
(para about authors)

Reply:
Thank you, I've split this now.